migration-engine/migration-engine-tests/tests/migrations/sql.rs

```diff
@@ -132,15 +132,19 @@ fn relations_to_models_with_no_pk_and_a_single_unique_required_field_work(api: T
         });
 }
 
-// TODO: Enable SQL Server when cascading rules are in PSL.
-#[test_connector(exclude(Mssql))]
-fn reserved_sql_key_words_must_work(api: TestApi) {
+#[test_connector]
+fn reserved_sql_keywords_must_work(api: TestApi) {
     // Group is a reserved keyword
     let dm = r#"
```

Comment:
For these you could use the new api.schema_push_w_datasource(dm1). This is adds the correct datasource and generator (generator should probably be mentioned in the method name). You'll probably get that already as merge conflict though.

Reply:
I'll do that in another review round later this week!